src/prefs/koPrefs.py

```diff
@@ -1571,6 +1571,7 @@ def __init__(self):
                     getService(components.interfaces.nsIObserverService)
         obsvc.addObserver(self, 'xpcom-shutdown', False)
         obsvc.addObserver(self, 'profile-before-change', False)
+        obsvc.addObserver(self, 'idle', False)
```

Comment:
As far as I can tell, this shouldn't have an effect.  I've not looked at this before but it looks like idle monitoring is handled by an underlying Mozilla service which is at line 1548.  Did you confirm that this changes anything in `def observe`?  I don't think this should be added.

Reply:
I did add debugging to `observer` and never saw an idle `topic`.  However, maybe I didn't wait long enough.  I removed it.

Reply:
You added the debugging before or after the change? I'd be happy to keep this if you tested before the change and saw no calls then tested after and saw action.  If there was no change let's keep it out.

Reply:
Yes.  I added it before and after.  It worked after, but not before, but I think I switched to the terminal to watch the log and it appears that Mozilla doesn't fire idle if you aren't focused on the app.  I realized that later and forgot to retest the original code.  I tested again and it looks like it's not needed.